drivers/flash/flash_stm32_ospi.c

```diff
@@ -79,6 +107,10 @@ struct flash_stm32_ospi_data {
 	enum jesd216_mode_type read_mode;
 	enum jesd216_dw15_qer_type qer_type;
 	int cmd_status;
+
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_dma) || DT_HAS_COMPAT_STATUS_OKAY(st_stm32_dmamux)
```

Comment:
Can't you use `STM32_OSPI_USE_DMA` here ?

Reply:
changed to #if STM32_OSPI_USE_DMA